Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM golang:1.6.2
+FROM golang:1.8-alpine
 MAINTAINER Sven Dowideit <SvenDowideit@home.org.au>
 
 ENV GOPATH /go
 ENV USER root
 
+RUN apk update && apk add git
+
 # pre-install known dependencies before the source, so we don't redownload them whenever the source changes
+# TODO can this list be generated?
```

Comment:
i want to use this Dockerfile for testing current code
(and to keep "make docker")
i'll add an commit
but maybe we want more Dockerfiles:
* one for clean build current github version
* one for fast build and test current $PWD version
* minimal image with latest stable huge version for becoming official dockerhub umage